test/integration/stylelint-vscode/__snapshots__/test.ts.snap

```diff
@@ -10,7 +10,7 @@ Array [
     "message": "Unexpected unknown at-rule \\"@unknown\\" (scss/at-rule-no-unknown)",
     "range": Object {
       "end": Object {
-        "character": 1,
+        "character": 30,
```

Comment:
Hmm. `scss/at-rule-no-unknown` seems to report the entire AT rule. Any ideas for a workaround?

```scss
@unknown (max-width: 960px) {}
```

Reply:
Oh, the `column` property isn't supported by `stylelint.utils.report()`... 😢 

https://github.com/stylelint-scss/stylelint-scss/blob/c27f7f2d1adffe86a2efc2010024238aeb0dc0c6/src/rules/at-rule-no-unknown/index.js#L87

https://github.com/stylelint/stylelint/blob/2c2a57cdca138aa6afe225b90b3fa6d2110245fc/lib/utils/report.js#L18

I have no idea to solve the problem, sorry.

Reply:
Thank you for the explanation. Do you think it can be considered a bug in scss/at-rule-no-unknown? If it's a bug, I think this change in the vscode extension can be considered fine.

Reply:
> Do you think it can be considered a bug in scss/at-rule-no-unknown?

Yes, I think so.

Reply:
> the column property isn't supported by stylelint.utils.report()

@ybiquitous `node.rangeBy` may also return columns in the same way as [lines](https://github.com/stylelint/stylelint/blob/2c2a57cdca138aa6afe225b90b3fa6d2110245fc/lib/utils/report.js#L35).

e.g. `const startColumn = column || (start && start.column);`

Reply:
> I would like plugin authors to use index / endIndex or word instead of line.

SGTM.